src/app/components/homePage/shared/time-selector.js

```diff
@@ -18,70 +18,75 @@ class TimeSelector extends React.Component {
       })
       return (
        <div className='time-picker'>
-         <h3>Submit All Availability Times:</h3>
+         <h3>Submit Time Slots For Available Times:</h3>
```

Comment:
@dylanlcole I don't agree with this message as opposed to the previous? Can we revert back please? Or do a different variation?